MarsOnboardingTask/StepDefinitions/LanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,112 @@
+using MarsOnboardingTask.Pages;
+using MarsOnboardingTask.Utils;
+using NUnit.Framework;
+
+
+namespace MarsOnboardingTask.StepDefinitions
+{
+    [Binding]
+    public class LanguagesStepDefinitions : Hooks
+    {
+        Login loginPageObj = new Login();
```

Comment:
please make use of constructors, it is best practice to make use of this when creating objects 
Please see 
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors